ui/tx/details/txDetailsActions/TxDetailsActionsFallback.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import type { TxAction } from 'types/api/txAction';
+
+import TxDetailsAction from './TxDetailsAction';
+import TxDetailsActionsWrapper from './TxDetailsActionsWrapper';
+
+interface Props {
+  actions: Array<TxAction>;
+  isLoading: boolean;
+}
+
+const TxDetailsActionsFallback = ({ actions, isLoading }: Props) => {
```

Comment:
The "fallback" suffix in the component name is a bit confusing. Could we use "TxDetailsActionsInterpreted" and "TxDetailsActionsRaw" instead? In my understanding, "fallback" is typically used to display something when an error occurs.